tools/rpm2img

```diff
@@ -219,18 +219,22 @@ dd if="${BOTTLEROCKET_DATA}" of="${DATA_IMAGE}" conv=notrunc bs=1M seek=1
 sgdisk -v "${DISK_IMAGE}"
 sgdisk -v "${DATA_IMAGE}"
 
+IMAGE_EXTENSION="img.lz4"
 if [[ ${OUTPUT_FMT} == "raw" ]]; then
   lz4 -vc "${DISK_IMAGE}" >"${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.img.lz4"
   lz4 -vc "${DATA_IMAGE}" >"${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.img.lz4"
-  chown 1000:1000 "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.img.lz4" \
-    "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.img.lz4"
+elif [[ ${OUTPUT_FMT} == "qcow2" ]]; then
```

Comment:
Rather than having an `elif` for each format, could we just do an `else`? `raw` and `vmdk` are special cases since `raw`'s extension ends up being `img.lz4` and `vmdk` requires some extra parameters, but other formats like `qcow2` and `vdi` could be in a more simple catch-all else statement like:

```
qemu-img convert -f raw -O "${OUTPUT_FMT}" "${DISK_IMAGE}" "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.${OUTPUT_FMT}"
qemu-img convert -f raw -O "${OUTPUT_FMT}" "${DATA_IMAGE}" "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.${OUTPUT_FMT}"
```
Users would still be limited to formats in the enum in `manifest.rs`, but we could reduce future code changes to just edge-cases here in `rpm2img`.

Reply:
I'd rather error out in the "else" branch than pick a format that wasn't requested and somehow we made it down here.

Reply:
I'm going to assume that the emojis win here.